enterprise/backend/src/metabase_enterprise/dependencies/api.clj

```diff
@@ -168,26 +168,119 @@
 (defn- format-subentity [entity]
   (case (t2/model entity)
     :model/Collection (select-keys entity [:id :name :authority_level :is_personal])
-    :model/Dashboard  (select-keys entity [:id :name])
-    :model/Document   (select-keys entity [:id :name])
+    :model/Dashboard (select-keys entity [:id :name])
+    :model/Document (select-keys entity [:id :name])
     entity))
 
-(defn- entity-value [entity-type {:keys [id] :as entity} usages]
-  {:id               id
-   :type             entity-type
-   :data             (->> (select-keys entity (entity-keys entity-type))
-                          (m/map-vals format-subentity))
-   :dependents_count (usages [entity-type id])})
+(mr/def ::usages
+  [:map-of
+   [:enum :table :snippet :transform :dashboard :document :sandbox :segment :question :model :metric]
+   :int])
```

Comment:
`pos-int?` instead of `:int`?

Might even be worth specifying `::entity-id` as a separate schema or something to make the purpose of `:int` clearer... otherwise it's not super clear what the values in `:usages` are (not just when looking at it, but in the generated dox either)